File: day01/python/part2.py
import time


def calculate(path):
    with open(path, "r") as f:
        lines = f.readlines()
        result = 0
        for line in lines:
            print(line)
        return str(result)


if __name__ == "__main__":
    start = time.time_ns()
    result = calculate("../input.txt")
    end = time.time_ns()
    if result[len(result) - 1] == "\n":
        result = result[: len(result) - 1]
    print(f"Result:\n{result}")
    print(f"Execution Time: {(end - start)/1000000:.3f} ms")

File: day07/bun/part2.js
export async function calculate(path) {
  const fullFileContents = await Bun.file(path).text()
  const lines = fullFileContents.trim().split("\n")

  let result = 0

  for (let i = 0; i < lines.length; i++) {
    const line = lines[i]

    // BEGIN TO DELETE
    result = line.length
    // END TO DELETE
  }

  return result.toString()
}

if (import.meta.main) {
  const start = performance.now()
  let result = await calculate("../input.txt")
  const end = performance.now()

  console.log(`Result:\n${result.trimEnd()}`)
  console.log(`Execution time: ${(end - start).toFixed(3)} ms`)
}
